tests/README.md

```diff
@@ -0,0 +1,138 @@
+# Testing
+
+This project uses the [helm-unittest project](https://github.com/quintush/helm-unittest) to test the helm charts included within. Once that plugin is installed, you can run tests with `helm unitest -3 .`
```

Comment:
We could also move to a more Pythonic way if helm-unittest needs to be forked  https://github.com/apache/airflow/blob/master/TESTING.rst#helm-unit-tests

WDYT? We use it for OSS Airflow and it has worked well for us so far, but I don't have a strong preference here.

Example commit to add a feature and add unit test for it in for OSS Airflow: https://github.com/apache/airflow/commit/441b4ef19f07d8c72cd38a8565804e56e63b543c#diff-f02a725253639ce892b023d3b6e307946244ebcba2b50af02feed402ca80ace9

Reply:
Interesting. I had thought about the possibility of testing helm with python but never looked into it. I'll have to think about this some more.

Reply:
I think this is a good idea. We should open another issue for it though.